docs/source/install.rst

```diff
@@ -1,42 +1,145 @@
 Installing Quipucords
 =====================
-Quipucords is delivered in two parts, a command line tool as an RPM package and a server as a container image. The following instructions describe how to install the parts of Quipucords.
+Quipucords is delivered in two parts, a command line tool as an RPM package and a server as a container image. The following instructions describe how to install the parts of Quipucords using an scripted installation or step by step commands.
+
+Scripted Installation
+----------------------
+The scripted installation utilizes Ansible to install the command line and container image and any associated dependencies.
+
+Installing the Ansible Prerequisite
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Follow the `Ansible installation documentation <http://docs.ansible.com/ansible/latest/intro_installation.html#installing-the-control-machine>`_ to prepare the your system to install the software.
```

Comment:
to prepare the your system --> to prepare the system

Reply:
done